src/components/StockTable/StockInfoTable.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import {Link, Table, Tbody, Td, Th, Thead, Tr, useColorModeValue} from "@chakra-ui/react";
+import {Link as ReactRouterLink} from "react-router-dom";
+
+const StockInfoTable = ({stocks}) => (
+    <Table variant="striped"
+           border='4px'
+           borderColor={useColorModeValue('brand.200', 'brand.600')}
+           colorScheme='brand'
+    >
+        <Thead>
+            <Tr>
+                <Th>Ticker Id</Th>
+                <Th>Dividend Payout Ratio</Th>
+                <Th display={{base: 'none', md: 'table-cell'}}>PE Ratio</Th>
+                <Th display={{base: 'none', md: 'table-cell'}}>PEG Ratio</Th>
+                <Th display={{base: 'none', lg: 'table-cell'}}>Quarterly Earning Growth</Th>
+                <Th display={{base: 'none', lg: 'table-cell'}}>Quarterly Revenue Growth</Th>
+                <Th>Fifty-two Week Low</Th>
```

Comment:
change this and stock-details page both to "52-week high" or something, keep it consistent (maybe use an enum?)